### DISCUSSION TITLE
posts/2023/06/leaving-google-domains-porkbun/

### DISCUSSION THREAD

**Discussion by giscus** [brycewray/comments#27] [Announcements]
# posts/2023/06/leaving-google-domains-porkbun/

An unexpected move by Big G prompts me to speed up an exodus I’d already considered.

https://www.brycewray.com/posts/2023/06/leaving-google-domains-porkbun/

**Comment by idarek** [comment]
Despite signing up for a deal between Google and Squarespace may meet struggles from regulators and the change will not start to happen, I got 3 domains used to have with them. Used, as with this unofficial announcement (Google do not notify customers), I moved 2 domains to CloudFlare where I noticed I will be paying even less than with Google (where I thought Google Domains were cheap). Sadly CloudFlare does not support every domain so one (.art) I moved elsewhere. It will be interesting to see how many domain will be left for the deal once is approved and finalised.
